canonical-web-and-design/charmhub.io
# Remove charm badges

## Done
- Remove framework badges
- Remove functionality badges

## How to QA
- Check cards are displayed fine without badges: https://charmhub-io-1048.demos.haus
- Check reactive charms are not showing the advert message: https://charmhub-io-1048.demos.haus/containers-metallb-controller

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2097
